Только в /root/dk_scripts/tmp/2: 6-Nov-2024015806LINK-I-Upgi10Ksw-sever-1-4_10.9.1.5.conf
Только в /root/dk_scripts/tmp/1: sw-sever-1-4_10.9.1.5.conf
diff -r /root/dk_scripts/tmp/2/TCS3-SW1-24_192.168.22.81.conf /root/dk_scripts/tmp/1/TCS3-SW1-24_192.168.22.81.conf
193,205d192
< interface fastethernet15
<  storm-control broadcast enable 
<  storm-control broadcast level 10 
<  storm-control include-multicast 
<  port security max 10 
<  port security mode max-addresses 
<  port security discard trap 60 
<  spanning-tree portfast 
<  switchport trunk allowed vlan add 2 
<  macro description ip_phone_desktop
<  !next command is internal.
<  macro auto smartport dynamic_type ip_phone_desktop 
< !
